### ISSUE TITLE
Build not emitting scripts included in web_accessible_resources

### ISSUE THREAD

**Issue by blake-regalia** [samrum/vite-plugin-web-extension#30]
For example, input Manifest V3:
```json
{
  "web_accessible_resources": [
    {
      "resources": ["src/inpage.ts"],
      "matches": ["file://*/*", "http://*/*", "https://*/*"]
    }
  ]
}
```

Makes it into the output `manifest.json`, but the transpiled version of `src/inpage.ts => src/inpage.{HASH}.js` is never emitted in the output (and the path in the manifest still points to the typescript file).

**Comment by blake-regalia** [comment]
I think we can close this issue once a new release is published. Any plans for a v1.1.0 release?

**Comment by samrum** [comment]
Yeah, there will be a release soon, thanks! There is just one manifest V3 issue around web accessible resources and match patterns that I want to fix first.

**Comment by soultice** [comment]
This seems to happen for `devtools_page` as well. Transpiled manifest includes the entry but associated html is not emitted.

**Comment by samrum** [comment]
Released as part of v2.0.0
